Panic after adding additional sensors

**Describe the bug**
I had previously the `cpu` sensors and then a fan and curve for each `cpu`, `front` and `rear`.

**To Reproduce**

After having had those initialized correctly I wanted to add the `gpu` sensor, fan, and curve. 
Doing that results in the panic below. 

**Expected behavior**
I want to simply add the `gpu` sensors, fan, and curve.

**Screenshots**
**Desktop (please complete the following information):**

- Distro: [e.g. Arch Linux]
- `uname -a`: `Linux metalmatze-desktop 6.18.2-2-cachyos #1 SMP PREEMPT_DYNAMIC Fri, 19 Dec 2025 01:23:45 +0000 x86_64 GNU/Linux`
- `sensors -v`:  `sensors version 3.6.2 with libsensors version 3.6.2`
- `fan2go version`: `0.11.1`

**Additional context**
```
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: WARNING: Fan 'front' has not yet been analyzed, starting initialization sequence...
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: INFO: Loading fan curve data for fan 'cpu'...
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: INFO: FanController: Using saved value for setPwmToGetPwmMap of Fan 'cpu'
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: INFO: Computing pwm map...
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: INFO: PWM settings of fan 'cpu': Min 24, Start 24, Max 251
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: INFO: Starting controller loop for fan 'cpu'
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: WARNING: Fan 'gpu' has not yet been analyzed, starting initialization sequence...
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: INFO: FanController: Using saved value for setPwmToGetPwmMap of Fan 'gpu'
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: INFO: Computing pwm map...
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: panic: ExpandMapToFullRange: inputControlPoints cannot be empty.
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: goroutine 120 [running]:
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal/util.ExpandMapToFullRange(0xc0001cd7f0, 0x0, 0xff)
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/util/math.go:88 +0x485
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).computePwmMapAutomatically(0xc0003320b0)
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/controller/controller.go:753 +0x468
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).computePwmMap(0xc0003320b0)
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/controller/controller.go:715 +0x256
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).computeFanSpecificMappings(0xc0003320b0)
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/controller/controller.go:845 +0x85
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).RunInitializationSequence(0xc0003320b0)
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/controller/controller.go:385 +0x3f
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).runInitializationIfNeeded(0xc0003320b0)
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/controller/controller.go:283 +0x196
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).Run(0xc0003320b0, {0x55bab1f9d930, 0xc000167ae0})
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/controller/controller.go:193 +0x16a
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/markusressel/fan2go/internal.RunDaemon.func7()
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/markusressel/fan2go/internal/backend.go:141 +0x3f
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: github.com/oklog/run.(*Group).Run.func1({0xc00048c140?, 0xc000482078?})
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/oklog/run@v1.1.0/group.go:38 +0x29
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]: created by github.com/oklog/run.(*Group).Run in goroutine 1
Dec 31 16:52:18 metalmatze-desktop fan2go[738535]:         github.com/oklog/run@v1.1.0/group.go:37 +0x5a
```

My current config:
```yaml
sensors:
- id: cpu
  hwmon:
    platform: k10temp-pci-00c3
    index: 1
- id: gpu
  hwmon:
    platform: amdgpu-pci-02900
    index: 1

fans:
- id: gpu
  hwmon:
    platform: amdgpu-pci-02900
    rpmChannel: 1
    pwmChannel: 1
  neverStop: true
  curve: gpu
- id: cpu
  hwmon:
    platform: nct6795-isa-0a20
    rpmChannel: 2
    pwmChannel: 2
  neverStop: true
  curve: cpu
- id: rear
  hwmon:
    platform: nct6795-isa-0a20
    rpmChannel: 3
    pwmChannel: 3
  neverStop: false
  curve: rear
- id: front
  hwmon:
    platform: nct6795-isa-0a20
    rpmChannel: 6
    pwmChannel: 6
  neverStop: false
  curve: case

curves:
- id: cpu
  linear:
    sensor: cpu
    min: 40
    max: 85
- id: rear
  linear:
    sensor: cpu
    min: 60
    max: 85
- id: gpu
  linear:
    sensor: gpu
    min: 40
    max: 70
- id: case
  function:
    type: average
    curves:
      - cpu
      - gpu
```

Tried the PR and saw this: 
```
[94m[94m███████ [0m[0m[94m[94m █████  [0m[0m[94m[94m███    ██ [0m[0m[37m[37m██████  [0m[0m[94m[94m ██████  [0m[0m[94m[94m ██████  [0m[0m
[94m[94m██      [0m[0m[94m[94m██   ██ [0m[0m[94m[94m████   ██ [0m[0m[37m[37m     ██ [0m[0m[94m[94m██       [0m[0m[94m[94m██    ██ [0m[0m
[94m[94m█████   [0m[0m[94m[94m███████ [0m[0m[94m[94m██ ██  ██ [0m[0m[37m[37m █████  [0m[0m[94m[94m██   ███ [0m[0m[94m[94m██    ██ [0m[0m
[94m[94m██      [0m[0m[94m[94m██   ██ [0m[0m[94m[94m██  ██ ██ [0m[0m[37m[37m██      [0m[0m[94m[94m██    ██ [0m[0m[94m[94m██    ██ [0m[0m
[94m[94m██      [0m[0m[94m[94m██   ██ [0m[0m[94m[94m██   ████ [0m[0m[37m[37m███████ [0m[0m[94m[94m ██████  [0m[0m[94m[94m ██████  [0m[0m

[30;46m[30;46m INFO [0m[0m [96m[96mVersion: 0.11.1[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mUsing configuration file at: /etc/fan2go/fan2go.yaml[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mGathering sensor data for cpu...[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mGathering sensor data for rear...[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mGathering sensor data for front...[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mGathering sensor data for gpu...[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mLoading fan curve data for fan 'cpu'...[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mLoading fan curve data for fan 'rear'...[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mLoading fan curve data for fan 'front'...[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mLoading fan curve data for fan 'gpu'...[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mFan 'cpu' has not yet been analyzed, starting initialization sequence...[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mFan 'front' has not yet been analyzed, starting initialization sequence...[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mFan 'gpu' has not yet been analyzed, starting initialization sequence...[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 0 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 1 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 2 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 3 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 4 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 5 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 6 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 7 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 8 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 9 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 10 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 11 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 12 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 13 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 14 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 15 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 16 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 17 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 18 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 19 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 20 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 21 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 22 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 23 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 24 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 25 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 26 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 27 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 28 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 29 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 30 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 31 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 32 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 33 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 34 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 35 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 36 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 37 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 38 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 39 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 40 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 41 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 42 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 43 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 44 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 45 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 46 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 47 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 48 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 49 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 50 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 51 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 52 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 53 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 54 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 55 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 56 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 57 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 58 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 59 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 60 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 61 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 62 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 63 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 64 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 65 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 66 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 67 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 68 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 69 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 70 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 71 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 72 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 73 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 74 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 75 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 76 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 77 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 78 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 79 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 80 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 81 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 82 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 83 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 84 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 85 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 86 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 87 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 88 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 89 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 90 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 91 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 92 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 93 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 94 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 95 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 96 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 97 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 98 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 99 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 100 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 101 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 102 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 103 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 104 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 105 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 106 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 107 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 108 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 109 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 110 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 111 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 112 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 113 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 114 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 115 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 116 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 117 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 118 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 119 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 120 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 121 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 122 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 123 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 124 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 125 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 126 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 127 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 128 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 129 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 130 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 131 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 132 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 133 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 134 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 135 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 136 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 137 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 138 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 139 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 140 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 141 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 142 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 143 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 144 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 145 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 146 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 147 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 148 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 149 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 150 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 151 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 152 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 153 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 154 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 155 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 156 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 157 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 158 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 159 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 160 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 161 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 162 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 163 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 164 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 165 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 166 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 167 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 168 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 169 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 170 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 171 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 172 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 173 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 174 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 175 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 176 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 177 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 178 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 179 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 180 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 181 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 182 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 183 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 184 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 185 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 186 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 187 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 188 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 189 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 190 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 191 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 192 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 193 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 194 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 195 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 196 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 197 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 198 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 199 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 200 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 201 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 202 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 203 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 204 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 205 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 206 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 207 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 208 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 209 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 210 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 211 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 212 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 213 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 214 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 215 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 216 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 217 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 218 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 219 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 220 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 221 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 222 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 223 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 224 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 225 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 226 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 227 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 228 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 229 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 230 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 231 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 232 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 233 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 234 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 235 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 236 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 237 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 238 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 239 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 240 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 241 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 242 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 243 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 244 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 245 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 246 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 247 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 248 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 249 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 250 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 251 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 252 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 253 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 254 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;43m[30;43m WARNING [0m[0m [33m[33mError setting PWM value 255 on fan gpu: write /sys/devices/pci0000:00/0000:00:03.1/0000:27:00.0/0000:28:00.0/0000:29:00.0/hwmon/hwmon2/pwm1: invalid argument[0m[0m
[30;46m[30;46m INFO [0m[0m [96m[96mComputing pwm map...[0m[0m
panic: ExpandMapToFullRange: inputControlPoints cannot be empty.

goroutine 182 [running]:
github.com/markusressel/fan2go/internal/util.ExpandMapToFullRange(0xc0003c97f0, 0x0, 0xff)
	github.com/markusressel/fan2go/internal/util/math.go:88 +0x485
github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).computePwmMapAutomatically(0xc0000ea630)
	github.com/markusressel/fan2go/internal/controller/controller.go:753 +0x468
github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).computePwmMap(0xc0000ea630)
	github.com/markusressel/fan2go/internal/controller/controller.go:715 +0x256
github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).computeFanSpecificMappings(0xc0000ea630)
	github.com/markusressel/fan2go/internal/controller/controller.go:845 +0x85
github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).RunInitializationSequence(0xc0000ea630)
	github.com/markusressel/fan2go/internal/controller/controller.go:385 +0x3f
github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).runInitializationIfNeeded(0xc0000ea630)
	github.com/markusressel/fan2go/internal/controller/controller.go:283 +0x196
github.com/markusressel/fan2go/internal/controller.(*DefaultFanController).Run(0xc0000ea630, {0x55906db68930, 0xc0000ff860})
	github.com/markusressel/fan2go/internal/controller/controller.go:193 +0x16a
github.com/markusressel/fan2go/internal.RunDaemon.func7()
	github.com/markusressel/fan2go/internal/backend.go:141 +0x3f
github.com/oklog/run.(*Group).Run.func1({0xc00040c100?, 0xc000402060?})
	github.com/oklog/run@v1.1.0/group.go:38 +0x29
created by github.com/oklog/run.(*Group).Run in goroutine 1
	github.com/oklog/run@v1.1.0/group.go:37 +0x5a
```

--- a/internal/controller/controller.go
+++ b/internal/controller/controller.go
@@ -3,12 +3,13 @@ package controller
 import (
 	"context"
 	"errors"
-	"github.com/markusressel/fan2go/internal/control_loop"
-	"golang.org/x/exp/maps"
 	"math"
 	"sort"
 	"sync"
 	"time"
+
+	"github.com/markusressel/fan2go/internal/control_loop"
+	"golang.org/x/exp/maps"
 
 	"github.com/markusressel/fan2go/internal/configuration"
 	"github.com/markusressel/fan2go/internal/curves"
@@ -677,6 +678,11 @@ func (f *DefaultFanController) computeSetPwmToGetPwmMap() (err error) {
 		return err
 	}
 
+	if f.setPwmToGetPwmMap == nil || len(maps.Keys(f.setPwmToGetPwmMap)) <= 0 {
+		ui.Warning("Fan '%s' setPwmToGetPwmMap is empty, ignoring", f.fan.GetId())
+		return nil
+	}
+
 	ui.Debug("Saving setPwmToGetPwmMap to fan...")
 	return f.persistence.SaveFanSetPwmToGetPwmMap(f.fan.GetId(), f.setPwmToGetPwmMap)
 }
@@ -816,7 +822,7 @@ func (f *DefaultFanController) computeSetPwmToGetPwmMapAutomatically() error {
 
 	_ = trySetManualPwm(f.fan)
 
-	f.setPwmToGetPwmMap = map[int]int{}
+	setPwmToGetPwmMap := map[int]int{}
 	for i := fans.MinPwmValue; i <= fans.MaxPwmValue; i++ {
 		err := f.fan.SetPwm(i)
 		if err != nil {
@@ -829,7 +835,12 @@ func (f *DefaultFanController) computeSetPwmToGetPwmMapAutomatically() error {
 			ui.Warning("Error reading PWM value of fan %s: %v", f.fan.GetId(), err)
 			continue
 		}
-		f.setPwmToGetPwmMap[i] = pwm
+		setPwmToGetPwmMap[i] = pwm
+	}
+
+	if len(setPwmToGetPwmMap) > 0 {
+		// we can only use the map if we have at least one entry
+		f.setPwmToGetPwmMap = setPwmToGetPwmMap
 	}
 
 	return nil

--- a/internal/persistence/persistence.go
+++ b/internal/persistence/persistence.go
@@ -4,12 +4,13 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/markusressel/fan2go/internal/fans"
-	"github.com/markusressel/fan2go/internal/ui"
-	bolt "go.etcd.io/bbolt"
 	"os"
 	"path/filepath"
 	"time"
+
+	"github.com/markusressel/fan2go/internal/fans"
+	"github.com/markusressel/fan2go/internal/ui"
+	bolt "go.etcd.io/bbolt"
 )
 
 const (
@@ -177,11 +178,15 @@ func (p persistence) LoadFanSetPwmToGetPwmMap(fanId string) (map[int]int, error)
 	defer func(db *bolt.DB) {
 		_ = db.Close()
 	}(db)
+	return p.loadFanSetPwmToGetPwmMap(db, fanId)
+}
 
+// loadFanSetPwmToGetPwmMap loads the "pwm requested" -> "actual pwm" map of the given fan from persistence
+func (p persistence) loadFanSetPwmToGetPwmMap(db *bolt.DB, fanId string) (map[int]int, error) {
 	key := fanId
 
 	var pwmMap map[int]int
-	err = db.View(func(tx *bolt.Tx) error {
+	err := db.View(func(tx *bolt.Tx) error {
 		b := tx.Bucket([]byte(BucketFanSetPwmToSetPwmMap))
 		if b == nil {
 			return os.ErrNotExist
@@ -204,6 +209,13 @@ func (p persistence) LoadFanSetPwmToGetPwmMap(fanId string) (map[int]int, error)
 
 		return err
 	})
+
+	if pwmMap != nil && len(pwmMap) <= 0 {
+		// remove invalid data from the db
+		fanId := fanId
+		_ = p.deleteFanSetPwmToGetPwmMap(db, fanId)
+		return nil, os.ErrNotExist
+	}
 
 	return pwmMap, err
 }
@@ -244,7 +256,10 @@ func (p persistence) DeleteFanSetPwmToGetPwmMap(fanId string) error {
 	defer func(db *bolt.DB) {
 		_ = db.Close()
 	}(db)
+	return p.deleteFanSetPwmToGetPwmMap(db, fanId)
+}
 
+func (p persistence) deleteFanSetPwmToGetPwmMap(db *bolt.DB, fanId string) error {
 	key := fanId
 
 	return db.Update(func(tx *bolt.Tx) error {

--- a/internal/persistence/persistence_test.go
+++ b/internal/persistence/persistence_test.go
@@ -129,6 +129,71 @@ func TestPersistence_LoadFanPwmData_SamplesNotInterpolated(t *testing.T) {
 	assert.Equal(t, expected, fanData)
 }
 
+func TestPersistence_LoadFanSetPwmToGetPwmMap_NotExisting(t *testing.T) {
+	// GIVEN
+	persistence := NewPersistence(dbTestingPath)
+
+	// WHEN
+	pwmMap, err := persistence.LoadFanSetPwmToGetPwmMap("non-existing-fan")
+
+	// THEN
+	assert.Nil(t, pwmMap)
+	assert.Error(t, err)
+}
+
+func TestPersistence_SaveAndLoadFanSetPwmToGetPwmMap(t *testing.T) {
+	// GIVEN
+	persistence := NewPersistence(dbTestingPath)
+
+	expectedPwmMap := map[int]int{
+		0:   0,
+		128: 130,
+		255: 255,
+	}
+
+	// WHEN
+	err := persistence.SaveFanSetPwmToGetPwmMap("fan1", expectedPwmMap)
+	assert.NoError(t, err)
+
+	pwmMap, err := persistence.LoadFanSetPwmToGetPwmMap("fan1")
+
+	// THEN
+	assert.NoError(t, err)
+	assert.Equal(t, expectedPwmMap, pwmMap)
+}
+
+func TestPersistence_SaveAndLoadFanSetPwmToGetPwmMap_NilSaveLoadsNil(t *testing.T) {
+	// GIVEN
+	persistence := NewPersistence(dbTestingPath)
+
+	// WHEN
+	err := persistence.SaveFanSetPwmToGetPwmMap("fan1", nil)
+	assert.NoError(t, err)
+
+	pwmMap, err := persistence.LoadFanSetPwmToGetPwmMap("fan1")
+
+	// THEN
+	assert.NoError(t, err)
+	assert.Nil(t, pwmMap)
+}
+
+func TestPersistence_SaveAndLoadFanSetPwmToGetPwmMap_EmptySaveLoadsNil(t *testing.T) {
+	// GIVEN
+	persistence := NewPersistence(dbTestingPath)
+
+	emptyMap := map[int]int{}
+
+	// WHEN
+	err := persistence.SaveFanSetPwmToGetPwmMap("fan1", emptyMap)
+	assert.NoError(t, err)
+
+	pwmMap, err := persistence.LoadFanSetPwmToGetPwmMap("fan1")
+
+	// THEN
+	assert.Nil(t, pwmMap)
+	assert.Error(t, err)
+}
+
 func createFan(neverStop bool, curveData map[int]float64) (fan fans.Fan, err error) {
 	configuration.CurrentConfig.RpmRollingWindowSize = 10
 

